docs/cookbook/ddd_architecture/basic_operations.md

```diff
@@ -0,0 +1,462 @@
+# Basic operations
+
+In previous chapter, we have created the Sylius resource. Now we need to create the basic operations.
+To achieve that, we reuse commands & queries we already have in the Application folder to create providers & processors.
+
+```txt
+src
+└── BookStore
+    ├── Application
+    │   ├── Command
+    │   │   ├── CreateBookCommand.php
+    │   │   ├── UpdateBookCommand.php
+    │   │   └── DeleteBookCommand.php
+    │   └── Query
+    │       └── FindBookQuery.php
+    ├── Domain
+    └── Infrastructure
+        └── Sylius
+            └── State
+                ├── Provider
+                │   └── BookItemProvider.php
+                └── Processor 
+                    ├── CreateBookProcessor.php   
+                    ├── UpdateBookProcessor.php
+                    └── DeleteBookProcessor.php
+```
+
+## Book creation
+
+<div data-full-width="false">
+
+<figure><img src="../../.gitbook/assets/create_book_resource.png" alt="Create book resource"></figure>
+
+</div>
+
+In the Application folder, we already have this `CreateBookCommand`:
+
+```php
+// src/Bookstore/Application/Command/CreateBookCommand.php
+
+declare(strict_types=1);
+
+namespace App\BookStore\Application\Command;
+
+use App\BookStore\Domain\Model\Book;
+use App\BookStore\Domain\ValueObject\Author;
+use App\BookStore\Domain\ValueObject\BookContent;
+use App\BookStore\Domain\ValueObject\BookDescription;
+use App\BookStore\Domain\ValueObject\BookName;
+use App\BookStore\Domain\ValueObject\Price;
+use App\Shared\Application\Command\CommandInterface;
+
+/**
+ * @implements CommandInterface<Book>
+ */
+final readonly class CreateBookCommand implements CommandInterface
+{
+    public function __construct(
+        public BookName $name,
+        public BookDescription $description,
+        public Author $author,
+        public BookContent $content,
+        public Price $price,
+    ) {
+    }
+}
+```
+
+The idea is to reuse this command to create the book in the storage for your "create" operation.
```

Comment:
I'm not sure how but I think this sentence should be rephrased to be less repetitive